sdk/core/azure-core/src/test/java/com/azure/core/util/logging/ClientLoggerTests.java

```diff
@@ -264,6 +266,20 @@ public void invalidLogLevelFromString(String environmentLogLevel) {
         assertThrows(IllegalArgumentException.class, () -> LogLevel.fromString(environmentLogLevel));
     }
 
+    @ParameterizedTest
+    @MethodSource("provideLogLevels")
+    @ResourceLock("SYSTEM_OUT")
+    public void logWithSupplier(LogLevel logLevel) {
+
+        setupLogLevel(logLevel.getLogLevel());
+        setPropertyToOriginalOrClear(Configuration.getGlobalConfiguration().get(PROPERTY_AZURE_LOG_LEVEL));
+        Supplier<String> supplier = () -> String.format("Param 1: %s, Param 2: %s, Param 3: %s", "test1", "test2", "test3");
+        logMessageWithInfoLevelForSupplier(new ClientLogger(ClientLoggerTests.class), logLevel, "", supplier);
+
+        String logValues = logCaptureStream.toString(StandardCharsets.UTF_8);
```

Comment:
`ByteArrayOutputStream.toString(Charset)` is only available in Java versions past Java 8 which is why CI is failing validation.

Reply:
Thanks, fixed!